src/App.tsx

```diff
@@ -1,8 +1,73 @@
-import React from 'react';
+import React, { Component } from 'react';
 import './App.css';
+import { GoodsList } from './components/GoodsList';
+import { getData } from './DATA_FROM_SERVER';
 
-const App = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+export class App extends Component {
+  state = {
+    goods: [],
+  };
 
-export default App;
+  loadGoods = () => {
+    getData()
```

Comment:
it is not general getData function but specific for goods function name it accordingly